kubehub/models/vm_group.py

```diff
@@ -8,9 +8,10 @@ class VMGroup(models.Model):
     name = models.CharField(max_length=name_max)
     user_id = models.IntegerField()
     readonly_fields = ('name', 'user_id')
+    status = models.CharField(max_length=name_max)
```

Comment:
add choices field to restrict status only to enum listed values

Reply:
fixed in a3a1558